checkQC/default_config/config.yaml

```diff
@@ -596,3 +596,80 @@ iseq_v1:
       - name: ReadsPerSampleHandler
         warning: unknown
         error: 3
+
+nextseq500_high:
```

Comment:
Did you leave out the ReadsPerSampleHandler for a specific reason? At our facility we set it as 0.75 * The expected Cluster PF. It is divided by the number of samples sequenced on the lane in the handler.

Reply:
The specific reason is called "overzealous copy and paste" ;) 
I was also trying to add support for the 2000 family before realizing it would need bclconvert support tacked in, so in the resulting "remove, rebase, etc." shuffle something got erased by mistake.
I'll add them back.